### DISCUSSION TITLE
[A-L] (14 Luglio 2023) Esercizio 2

### DISCUSSION THREAD

**Discussion by FeddyLix17** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#393] [Esami A-L]
Siano $\Large f\ :\ X \rightarrow Y$ e $\Large g\ :\ Z \rightarrow Y$ dove $\Large Z \subseteq X$. Indicare se le seguenti affermazioni sono vere o false.

NB: per un qualunque $\Large S \subseteq X$ con $\Large f(S)$ si denota l’insieme  $\Large \\{y \in Y\ :$ per qualche $\Large s \in S$ vale $\Large f(s) = y\\}.$

Analogamente per $\Large g(S)$.

1. Se $\Large f$ è iniettiva allora $\Large g$ è iniettiva. [1 punto]

2. $\Large f(X − Z) \subseteq f(X) − f(Z)$. [2 punti]

3. $\Large Y = f(X) \cup g(Z)$. [2 punti]

**Comment by FeddyLix17** [comment]
Tutte e 3 le affermazioni sono false

**Reply by M1SC0** [reply]
Penso che il tuo terzo punto non sia corretto (in caso fosse sbagliato il mio ragionamento rispondimi, grazie):
se la funzione fosse suriettiva, questo implicherebbe che ogni y in Y sia immagine di almeno una x in X e questo potrebbe effettivamente validare che f(X) U g(Z) = Y...ma non abbiamo la certezza che f(x) sia suriettiva. 
Quindi non avendone la certezza, non è detto che tutti gli elementi y in Y siano immagine di almeno una x in X, rendendo la risposta FALSA.

**Reply by cek753** [reply]
Penso che il primo punto sia sbagliato, non è detto che se f sia iniettiva allora anche g lo sia. Dipende dalla definizione di f e g: 
f(x)=x+1  (iniettiva)
g(x)=x^2 (suriettiva)
questo dovrebbe rendere la numero uno Falsa.

**Reply by FeddyLix17** [reply]
@cek753 $\Large g(x) = x^2$ non è suriettiva nei casi in cui l'insieme $\Large Y$ presenti dei valori negativi al suo interno.

Sicuramente però non è neanche iniettiva, il che rende la prima affermazione dell'esercizio falsa.

**Reply by ShallKnowMikan** [reply]
Il secondo punto penso sia vero solo se sappiamo che f e' iniettiva.
Ipotizziamo di avere f NON iniettiva.
Sia X = {1, 2 , 3} 
Sia Z = { 2 }
Sia Y = {A, B}

con: 
f(1) = A
f(2) = B
f(3) = B

X - Z = { 1, 3}
Quindi f(X - Z) = { A , B }

f(X) = {A , B}
f(Z) = {B}
f(X) - f(Z) = {A}
{A} != { A , B }

Siccome il testo non esclude la possibilita' che non sia iniettiva non possiamo darla per vera.
